examples/map/image_bright_regions_gallery_example.py

```diff
@@ -1,56 +1,36 @@
 # coding: utf-8
 """
-====================================================
-Manipulating Map image data - finding bright regions
-====================================================
+===================================
+Finding bright regions with ndimage
+===================================
 
-This example shows how you can do basic image processing on SunPy map image data.
-In this example, we try to find the brightest regions in an AIA image.
-Then count the approximate number of regions of interest.
+How you can to find the brightest regions in an AIA image and
+count the approximate number of regions of interest using ndimage.
 """
+# sphinx_gallery_thumbnail_number = 2
 
-##############################################################################
-# First, import the modules we will need:
 from scipy import ndimage
 import matplotlib.pyplot as plt
 
 import sunpy.map
 from sunpy.data.sample import AIA_193_IMAGE
 
-##############################################################################
-# Now, we create a SunPy Map object from an AIA FITS file.
-
+###############################################################################
+# We start with the sample data
 aiamap = sunpy.map.Map(AIA_193_IMAGE)
+aiamap_orig = sunpy.map.Map(AIA_193_IMAGE)
```

Comment:
errrr?

Reply:
We should never modify a map in place, so we really shouldn't have to do this.

Reply:
but we do in this example...I didn't even think we could!